include/boost/url/error_type.hpp

```diff
@@ -80,6 +121,12 @@ namespace urls {
 template<class T>
 using result = boost::system::result<T, error_code>;
 
+#ifndef BOOST_URL_DOCS
+} // error_types
```

Comment:
Yeah, this namespace is nice because I can just lift it into each downstream library (HTTP-Proto, HTTP-IO and so on)

Reply:
Nice 🙂